src/github.ts

```diff
@@ -143,6 +143,8 @@ interface ReleaseHistory {
 interface CommitIteratorOptions {
   maxResults?: number;
   backfillFiles?: boolean;
+  // if set, throw an error if we require a response
+  requireResponse?: boolean;
```

Comment:
Should we just always be throwing after too many failures? Wondering if we need to make this configurable.

Reply:
Ok, configured it to retry 5 times by default with exponential backoff (max of 20 seconds).